fiftyone/core/utils.py

```diff
@@ -1992,3 +1995,26 @@ def validate_hex_color(value):
 
 
 fos = lazy_import("fiftyone.core.storage")
+
+
+def load_dataset(identifier):
```

Comment:
By convention, things in `fiftyone.core.utils` are things that users may directly call. I would argue that users should not be loading datasets by `ObjectID` since this is not information that we publicly expose to them.

Can you please move this utility to the `fiftyone.core.odm.utils` module? That's a better place for internal-only utilities.

Reply:
sure I didn't intend this to be user facing but I didn't know which util file to use